spotbugs/src/main/java/edu/umd/cs/findbugs/AbstractBugReporter.java

```diff
@@ -35,6 +34,7 @@
 
 import org.dom4j.DocumentException;
 
+import edu.umd.cs.findbugs.annotations.Nullable;
```

Comment:
FindBugs `@Nullable` annotation doesn't mean anything, should use `@CheckForNull`

Reply:
Do we need to use it even for arguments? We accept null value then I think `@Nullable` is enough and intuitive.